app/scripts/templates/pair/success.mustache

```diff
@@ -0,0 +1,15 @@
+<div id="main-content" class="card pair-auth pair-auth-complete">
+  <header>
+    <h1 id="pair-auth-complete-header">{{#t}}Device connected{{/t}}</h1>
```

Comment:
https://www.lucidchart.com/documents/view/9a420c19-1e92-42a5-8eae-908a442c1044/7 

Simple success landing point, and r-b or fenix can choose to use this once redirected from native rust call.